charts/jx-app-jacoco/templates/service.yaml

```diff
@@ -1,23 +0,0 @@
-apiVersion: v1
-kind: Service
-metadata:
-{{- if .Values.service.name }}
```

Comment:
There is still a `service` section in `values.yaml` which can be deleted.

Reply:
I changed the service section - https://github.com/jenkins-x-apps/jx-app-jacoco/blob/master/charts/jx-app-jacoco/values.yaml#L9

I left the remaining properties since the HTTP server for now still exists. Still a bit undecided on this. Tempted to make it a proper health check.